Sources/ScipioKit/Logger.swift

```diff
@@ -68,26 +68,26 @@ struct ScipioLogHandler: LogHandler {
 }
 
 extension LoggingSystem {
-    public static func bootstrap() {
+    public static func bootstrap(logLevel: Logger.Level) {
         self.bootstrap { _ in
-            ScipioLogHandler()
+            var handler = ScipioLogHandler()
+            handler.logLevel = logLevel
+            return handler
         }
     }
 }
 
-func setLogLevel(_ level: Logger.Level) {
-    logger.logLevel = level
-}
-
-let observabilitySystem = ObservabilitySystem { _, diagnostics in
-    switch diagnostics.severity {
-    case .error:
-        logger.error("\(diagnostics.message)")
-    case .warning:
-        logger.warning("\(diagnostics.message)")
-    case .info:
-        logger.info("\(diagnostics.message)")
-    case .debug:
-        logger.debug("\(diagnostics.message)")
+func generateObservabilitySystem() -> ObservabilitySystem {
```

Comment:
`generate` is a bit unnatural on Swift language for making an instance.
And I doubt this function should be located at global level.

How about this?
```swift
extension ObservabilitySystem {
    static func makeForScipio() -> ObservabilitySystem {
        Observability { .... }
    }
}
```

Reply:
It's a bit in-convenient because callers must imports `Basics` module. It sometimes occurs conflicts of some symbols.

Then I renamed it to `makeObservabilitySystem`